src/install/local/prerequisites.md

```diff
@@ -0,0 +1,28 @@
+# Prerequisites
+----
+### System Dependencies
+Currently, our documentation is tailored for users operating on Linux or Mac computers. Additionally, it assumes you have the following:
+  - You have <a href="https://brew.sh/" target="_blank">🍺 homebrew</a> installed.
+  - You’re using __bash__, but you can modify these configurations and commands as needed for your preferred command line interface.
+  - If you're using MacOS, you'll also need <a href="https://mac.install.guide/commandlinetools/" target="_blank">XCode Commandline Tools</a> installed.
+  - You will also need to have <a href="https://kubernetes.io/releases/download/" target="_blank">Kubernetes</a> installed.
+
+</br >
+
+----
+### Note on user requirements
+We aim to make our solutions accessible to everyone. However, we recommend having an intermediate understanding of OpenTelemetry and Kubernetes, or a strong determination, before proceeding with this installation."
```

Comment:
Didn't like grit? 😝

Reply:
I have enough grit, so much so that I have to shower 3 times a day. :P  (mostly to prevent jungle rot from setting in)